tuner/tuner/libtuner.py

```diff
@@ -390,6 +407,18 @@ def parse_arguments() -> argparse.Namespace:
         action="store_true",
         help="Do not attempt to run any modules or initialize the IREE runtime",
     )
+    general_args.add_argument(
+        "--benchmark-out-format",
+        choices=[x.value for x in OutputFormats],
+        default=OutputFormats.json,
+        help="The benchmark output file format",
+    )
+    general_args.add_argument(
+        "--benchmark-out-path",
+        type=str,
+        default="bm_output.json",
+        help="The benchmark output file path/name",
+    )
```

Comment:
Should these be renamed to contain `_` instead of `-`?
I've noticed that they are named using `_` in other parts of the project

Reply:
We shouldn't add flag for this and instead use the right defaults